test/mpi/util/mtest_common.c

```diff
@@ -22,12 +22,12 @@ MPI_Aint MTestDefaultMaxBufferSize()
     char *envval = NULL;
     envval = getenv("MPITEST_MAXBUFFER");
     if (envval) {
-        max_size = atol(envval);
-        if (max_size < 100000) {
-            fprintf(stderr, "MPITEST_MAXBUFFER (%s) is too small, we need at least 100MB\n",
-                    envval);
+        MPI_Aint env_max_size = atol(envval);
+        if (env_max_size > max_size) {
+            fprintf(stderr, "MPITEST_MAXBUFFER (%s) is too large\n", envval);
```

Comment:
Isn't `MPITEST_MAXBUFFER` meant for cases where the user wants to use a larger buffer size?  This error seems to defeat that purpose.

Reply:
Right, I think I may have got it backwards. My interpretation was that if the user wants to change the max buffer size this should not exceed the default max_size. I will remove this commit